core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -800,7 +802,7 @@ private Name() {} // prevent instantiation
    * @return whether the input is a valid property name
    */
   public static boolean isValid(String keyStr) {
-    return KEYS_MAP.containsKey(keyStr);
+    return KEYS_MAP.containsKey(keyStr) || keyStr.startsWith("fs");
```

Comment:
@calvinjia @apc999 can you please tell me how to proceed with it? I need to load properties by prefix... if this is fine, i will need to update unitests.. or implement other way if you prefer something else